src/features/mcda/components/MCDALayerEditor/MCDALayerParameters/MCDALayerParameterRow/MCDALayerParameterRow.tsx

```diff
@@ -0,0 +1,31 @@
+import { TooltipTrigger } from '~components/TooltipTrigger';
+import { LAYERS_PANEL_FEATURE_ID } from '~features/layers_panel/constants';
+import s from './MCDALayerParameterRow.module.css';
+
+export type MCDALayerParameterRowProps = {
+  name: string;
+  tipText?: string;
+  children?: JSX.Element | JSX.Element[];
+};
+
+export function MCDALayerParameterRow({
+  name,
+  tipText,
+  children,
+}: MCDALayerParameterRowProps) {
+  return (
+    <div className={s.inputLine}>
+      <span className={s.inputLinelabel}>
+        {name}
+        {tipText && (
+          <TooltipTrigger
+            className={s.infoButton}
+            tipText={tipText}
+            tooltipId={LAYERS_PANEL_FEATURE_ID}
```

Comment:
This option is not required

Reply:
Reused tooltipId here by analogy from TooltipTrigger used in `LayerInfo`. My guess is it could be a part of some bugfix